src/doc/html.md

```diff
@@ -176,11 +176,11 @@ included.
 We use `analytics.js` rather than the newer `gtag.js` as
 [it's faster and supports tasks and plugins](https://github.com/philipwalton/analyticsjs-boilerplate/issues/19#issuecomment-333714370)
 
-Starting with version 8 we, by default, <a href="https://support.google.com/analytics/answer/2763052?hl=en">anonymizse IP addresses</a>.
+Starting with version 8 we, by default, [anonymize IP addresses](href="https://support.google.com/analytics/answer/2763052?hl=en).
 By default Google Analytics records the full IP address of a user visiting the site, but
 that full IP address is never available to the Google Analytics property admin.
 By anonymizing the IP address you can make your site more GDPR-compliant as a
-full IP address can be defined as PII (publicly identifiable information.)
+full IP address can be defined as PII (personally identifiable information.)
 
```

Comment:
sorry, last nit-pick and then we're good. There's 2 line breaks after this section and should be just 1 to be consistent.

Reply:
nit-picking is why we're here 👍